### PR TITLE
Added the Terraform Workspace Variables Operations

### PR DESCRIPTION
**Pull request by MeghanaManjunath98**
Added the Terraform Workspace Variables Operations.
Tested locally, everything works fine.

### CODE REVIEW

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+import static io.cloudslang.content.hashicorp.terraform.utils.Outputs.CreateVariableOutputs.VARIABLE_ID;
+import static io.cloudslang.content.hashicorp.terraform.utils.Outputs.CreateWorkspaceOutputs.WORKSPACE_ID;
+import static io.cloudslang.content.httpclient.entities.HttpClientInputs.*;
+import static io.cloudslang.content.utils.OutputUtilities.getFailureResultsMap;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+public class CreateWorkspaceVariable {
+
+    @Action(name = CREATE_VARIABLE_OPERATION_NAME,
```
Comment (patmante) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java:63]: Change the operation name to CREATE_WORKSPACE_VARIABLE_OPERATION_NAME,
Do similar changes to other operations also

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+            description = CREATE_VARIABLE_DESC,
```
Comment (patmante) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java:64]: Change description of the operation

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java (hunk 1 above)
Comment (MeghanaManjunath98) [reply cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java]: Changes have been made

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java (hunk 2 above)
Comment (MeghanaManjunath98) [reply cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/workspaces/variables/CreateWorkspaceVariable.java]: Changes have been made

**Pull request merged**